open_event/static/js/jquery/jquery.ellipsis.js

```diff
@@ -0,0 +1,42 @@
+// Borrowed from http://stackoverflow.com/a/1022672/1562480
+(function ($) {
+    $.fn.ellipsis = function () {
+        return this.each(function () {
+            var el = $(this);
+
+            var text = el.attr("data-original-text");
+            el.text(text);
+
+            var multiline = true;
+            
+            var t = $(this.cloneNode(true))
+                    .hide()
+                    .css('position', 'absolute')
+                    .css('overflow', 'visible')
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2650058367/issues/source?bid=3233864&fileBranchId=3324174#l15)